Environment/Environment.tex

```diff
@@ -620,27 +628,53 @@ \subsection{Packages: declarative categorization of Squeak code}
 \end{code}
 
 \noindent
-These methods belong in \damien{don't we say 'belong to'?} the same package as the classes in the categories \ct{SqueakLink-Connections} and \ct{SqueakLink-Model}.
+These methods belong in the same package as the classes in the categories \ct{SqueakLink-Connections} and \ct{SqueakLink-Model}.
 But clearly the whole of class \ct{Object} does not belong in your package!
 So you need a way of putting certain \emph{methods} in a package, even though the rest of the class is in another package.
 \index{package!extension}
-\seeindex{extension package}{package, extension}
+\seeindex{extension method}{package, extension}
+
+This is where the concept of \ind{extension methods} comes in.
+You can mark these converter methods as extension methods by placing them in a protocol (of \ct{Object}, \ct{String}, \ct{Date}, and so on) named \prot{*SqueakLink-Model-converting} (note the initial asterisk).
+% In general, any method in any protocol that starts with an asterisk, followed by a package name, followed by a hyphen, is an extension method to that package.

```

Comment:
I like this repetition of the more general rule. :)

Reply:
```suggestion
In general, any method in any protocol that starts with an asterisk, followed by a package name, followed by a hyphen, is an extension method to that package.
```